flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkDynamicTableFactory.java

```diff
@@ -142,6 +143,12 @@ public Set<ConfigOption<?>> optionalOptions() {
     Set<ConfigOption<?>> options = Sets.newHashSet();
     options.add(CATALOG_DATABASE);
     options.add(CATALOG_TABLE);
+    options.add(FlinkLookupOptions.LOOKUP_CACHE_MAX_ROWS);
+    options.add(FlinkLookupOptions.LOOKUP_CACHE_TTL);
+    options.add(FlinkLookupOptions.LOOKUP_CACHE_IGNORE_EMPTY);
+    options.add(FlinkLookupOptions.LOOKUP_MAX_RETRIES);
+    options.add(FlinkLookupOptions.LOOKUP_BASE_RETRY_BACKOFF_MILLS);
+    options.add(FlinkLookupOptions.LOOKUP_MAX_RETRY_BACKOFF_MILLS);
```

Comment:
Usually, we implement new features against just the latest version (in this case, Flink 1.14).

This makes reviewing easier as there isn't code duplication and comments aren't split across two sets of files.

Then, after everything is reviewed and merged, we back port to previous versions.

Is it possible to keep this just to Flink 1.14 for now?

Reply:
Accepted, please check again